storybook/manager-head.html

```diff
@@ -0,0 +1,35 @@
+<script>
+	( function redirectIfStoryMoved() {
+		const REDIRECTS = [
```

Comment:
Nit: Seems like this could be declared a single time outside of the function (while still kept private from the global scope in a closure)?

Reply:
Like in an additional wrapper function? Not exactly sure what you had in mind here.

Though I do think we should extract this into a separate script file if things get more complicated, or if we want to put more JS in this template part file.

Reply:
Yeah, an additional wrapper function. Fine to leave it as-is, this is quite a nitpick at this level.